longest_common_subsequence/longestCommonSubsequence.java

```diff
@@ -0,0 +1,44 @@
+
+class LongestCommonSubsequence
```

Comment:
This repo uses the "same line block opening" rule - 
```java
class LongestCommonSunsequence {
    ...
}
```
Please make this change in all instances.